### DISCUSSION TITLE
[01] Projektstart

### DISCUSSION THREAD

**Discussion by lukashert** [Chrissi-Ruege/HealthKeeper#3] [General]
# HealthKeeper

## 1. Team

| Name      | Rolle               |
|-----------|---------------------|
| Ekaterina | Frontend Entwickler |
| Lukas M.  | Frontend Entwickler |
| Christine | Backend Entwickler  |
| Lukas H.  | Backend Entwickler  |

## 2. Ziel

> Im Zuge des Projektes soll eine Fitnesstracking-App entwickelt werden. Es soll möglich sein verschiedene Daten rund um das Thema Fitness und Gesundheit innerhalb dieser App zu speichern und zu visualisieren. 

## 3. Geplanter Techstack

Um die App auf möglichst jeder Plattform nutzen zu können wird eine Webseite entwickelt. Hierzu wird ein C# basierter Techstack verwendet: 

- [ASP.NET](https://dotnet.microsoft.com/en-us/apps/aspnet) 
- [EntityFramework-Core](https://learn.microsoft.com/de-de/ef/core/)
- [JIRA](https://www.atlassian.com/de/software/jira)
- [GitHub Actions](https://github.com/features/actions)
- HTML / CSS / JavaScript

Als VCS wird [diese](https://github.com/Chrissi-Ruege/HealthKeeper) GitHub Repository verwendet.

## 4. Scrum Setup
Die Projektplanung findet in JIRA statt. Ein Sprint dauert 2 Wochen und beginnt immer Dienstags. Zu Beginn eines Sprints finden Review, Retro und Planning in einem Meeting von 14-16 Uhr statt.
Das Daily findet Montags und Donnerstags statt.

![Scrum Board](https://github.com/Chrissi-Ruege/HealthKeeper/assets/20227840/bd26bee5-a6cd-49fa-8daf-745b05c41de0)
![Backlog](https://github.com/Chrissi-Ruege/HealthKeeper/assets/20227840/34038e4f-a26a-4e3e-bce9-503a72f7b92d)

**Comment by Puggingtons** [comment]
Coole Projektidee! Bin gespannt, was dabei rauskommt. Habt ihr euch schon gedanken darüber gemacht, welche Daten gespeichert werden, bzw. in welcher Richtung im Fitness der Focus liegen wird?

**Comment by Fefke** [comment]
Klingt cool, habt ihr vor den Umfang weiter einzugrenzen? Ich denke rein Fitness und Gesundheit kann schon recht komplex werden,
vielleicht lässt das ausgereifte ASP.net eine schnellere Entwicklung zu - idk 🪟🧑‍💻^^.
In jedem Fall würde ich euch Empfehlen im Frontend ein UI Component Framework/Bibliothek, wie Tailwind Elements oder vielleicht sogar Bootstrap 5 einzusetzen, kann gut Nerven beim Design sparen und das Ergebnis ist konsistent.
Bin hyped auf eure Umsetzung 🥳

**Comment by Phylyp1** [comment]
Sehr gute Projektidee. Bin gespannt auf die Umsetzung 
LG Team School-Planner
